sklearn/manifold/_t_sne.py

```diff
@@ -656,9 +663,9 @@ class TSNE(BaseEstimator):
 
     @_deprecate_positional_args
     def __init__(self, n_components=2, *, perplexity=30.0,
-                 early_exaggeration=12.0, learning_rate=200.0, n_iter=1000,
+                 early_exaggeration=12.0, learning_rate="warn", n_iter=1000,
                  n_iter_without_progress=300, min_grad_norm=1e-7,
-                 metric="euclidean", init="random", verbose=0,
+                 metric="euclidean", init="warn", verbose=0,
```

Comment:
You need to update the docstring for `init` as well.

Reply:
Update how? You mean the docstring should mention that the default is going to change in version 1.2? Otherwise the current docstring is fine.

Reply:
Yes, I would mention the future default.

Reply:
OK -- added.